### PR TITLE
fix(django): fix broken iframe and add pysus to the API tutorial

### PR DESCRIPTION
**Pull request by luabida**
depends on https://github.com/AlertaDengue/PySUS/pull/127

### CODE REVIEW

File: containers/compose.yaml [hunk 1]
```diff
@@ -32,7 +32,6 @@ services:
       interval: 10s
       timeout: 5s
       retries: 10
-    command: -p ${PSQL_PORT}
```
Comment (luabida) [inline containers/compose.yaml:35]: @esloch found a duplicity here, is it ok to remove? It was breaking the docker-compose command

File: containers/compose.yaml [hunk 2]
```diff
@@ -57,7 +56,6 @@ services:
...
```
Comment (luabida) [inline containers/compose.yaml:60]: here too

**Comment by luabida** [comment]
@esloch I think the nbconvert won't create a frame as it was, it looks like a static html page now: https://infostaging.dengue.mat.br/services/api/doc

**Comment by esloch** [comment]
> @esloch I think the nbconvert won't create a frame as it was, it looks like a static html page now: https://infostaging.dengue.mat.br/services/api/doc

@luabida. I appreciate you working on this. Just keep in mind the template tags that Django uses for formatting, translation, and page layout. If you remove them, the page will behave unexpectedly. It's worth taking a look at the Django documentation.

**Comment by luabida** [comment]
@esloch ready to last review and merge

**Comment by esloch** [comment]
The proposed changes are acceptable to me, and I will proceed with the merge.
Thank you @luabida!

**Review by esloch — APPROVED** [review]
LGTM!

**Pull request merged**